### ISSUE TITLE
Bad error message when saving stl from an SR window

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#2581]
Run visit in parallel (2-4 procs)
Open multi_ucd3d.silo
Turn scalable rendering (SR) to always.
Put up a plot, say mesh plot
Draw it.
Go to Save Window Options
Select STL format.
Hit Save.
You get an error message saying that you cannot save Curve formats (curve, ultra) from an SR window.
I wasn't trying to save a curve format. I was trying to save a 3D format.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Mark Miller
Original creation: 06/30/2015 08:24 pm
Original update: 08/25/2015 07:02 pm
Ticket number: 2325
